load/UpdateDBPTextFilesets.py

```diff
@@ -54,39 +51,63 @@ def validateFileset(self, subTypeCode, bibleId, filesetId, languageRecord, lptsI
 		elif subTypeCode == "text_json":
 			self.newFilesetId = filesetId.split("-")[0] + "-json"	
 
+		return None
+
+	def invokeBiblePublisher(self, inp, fullFilesetPath):
```

Comment:
@bradflood , I have created this new method to separate the BiblePublisher logic from validateFileset method.